### PR TITLE
Circle CI: Avoid staticfloat's julia PPA on trusty

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes https://github.com/coala/coala/issues/2826
Fixes https://github.com/coala/coala-bears/issues/859

### CODE REVIEW

**Comment by jayvdb** [comment]
I am not sure what dependency from `ppa:staticfloat` julia is causing the problem on Trusty, so it is a bit difficulty to fill in the commit message with something meaningful, but I will improve it.  However, gotta run atm.

**Comment by jayvdb** [comment]
Note this is sufficient to build on precise and trusty. But Im rebuilding on both without cache before getting too excited.

**Comment by jayvdb** [comment]
Rebuilds without cache on precise and trusty are green.

**Comment by sils1297** [comment]
ack https://github.com/coala/coala/pull/2827/commits/e1ec25773851184bf43ce3a309eb078f2a2629c5

**Comment by jayvdb** [comment]
Note: I think I have another better solution coming.

**Comment by sils1297** [comment]
@jayvdb any news on the better solution?

**Comment by jayvdb** [comment]
I was hoping https://github.com/coala/coala-bears/issues/869 could be fixed before this `coala` fix, so they can both switch to trusty.
But that may take a few more days, so this can be merged if you want trusty `coala` builds to work now.
I haven't looked at the `coala` trusty build elapsed times, so switching the main repo builds to trusty may not be wise.

**Comment by jayvdb** [comment]
I was hoping https://github.com/coala/coala-bears/issues/869 could be fixed before this `coala` fix, so they can both switch to trusty.
But that may take a few more days, so this can be merged if you want trusty `coala` builds to work now.
(Feel free to rebase it if you need this urgently; im limited avail. for next two days.)
I haven't looked at the `coala` trusty build elapsed times, so switching the main repo builds to trusty may not be wise.

**Comment by sils** [comment]
argh the regex triggers because there's no space after the colon which is a common mistakes from newcomers.

**Comment by jayvdb** [comment]
no worries; im revising it anyway.

**Comment by jayvdb** [comment]
@sils , hmm.  hit the regex again.  This time, I think the regex that needs fixing.

**Comment by sils** [comment]
Just rephrase it to `staticfloat ppa` instead of doing it with the colon or change the regex. Important: make a commit to the coafile *before* the commit that violates the current regex. GitMate checks every commit individually, not only the PR.

**Comment by sils** [comment]
```diff
-shortlog_regex = ([^:]*|\S+: [A-Z0-9*].*)
+shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
```

should do it. We need to do that on all repos though...

**Comment by sils** [comment]
I added the coafile commit to https://github.com/coala/coala/pull/2873 to speed this up a bit since it seems important and easy
